### DISCUSSION TITLE
Vue ref and $ref are confusing for beginners.

### DISCUSSION THREAD

**Discussion by yogeshgalav** [vuejs/rfcs#487] [Polls]
I love the way Vue 3 feels like plain javascript,
But the reactivity thing is always hard to explain to beginners,
It's even hard to adapt for experienced vue 2 developers.

And what's more concerning is that new reactivity transform is even more difficult.
Please support if you want to make Vue simpler and easier.
https://github.com/vuejs/rfcs/discussions/485

**Comment by darrylnoakes** [comment]
The reactivity transform has been shelved and will not be included in core. A plugin will be made available for those who want to use it.

**Reply by yogeshgalav** [reply]
Even the basic ref is confusing in beginning, hence I made a suggestion which would be simpler to implement and awesome to use.
https://github.com/vuejs/rfcs/discussions/485

**Comment by Demivan** [comment]
Please don't spam discussions. You have already created https://github.com/vuejs/rfcs/discussions/485 so this discussion can be done there.

Yes `$ref` is confusing and that is why Reactivity transform was not included in core Vue.

**Reply by yogeshgalav** [reply]
Sorry sir, I just wanted to gain attention on new way of writing reactive variables,
I feel it's simpler and technically feasible.
